orion_integration/tasks.py

```diff
@@ -45,7 +45,9 @@ def populate_from_orion():
     return ret
 
 
-@shared_task(task_serializer='pickle', result_serializer='pickle')
+@shared_task(
+    task_serializer='pickle',
```

Comment:
Using  pickle for  Celery task serializer may open up to some security concerns. You can mitigate them, but it’s best to  tell Celery to use the JSON serializer:
CELERY_TASK_SERIALIZER = 'json'

Reply:
granted but i think we can live with pickle here considering how much work it will be to change the functions to support json serialization. there is also a performance hit when using json